app/build.gradle.kts

```diff
@@ -16,8 +17,8 @@ android {
         applicationId = "app.revanced.manager"
         minSdk = 26
         targetSdk = 34
-        versionCode = 1
-        versionName = "0.0.1"
+        versionName = project.version.toString()
+        versionCode = (majorVersion.toInt() * 100000000) + (minorVersion.toInt() * 100000) + (patchVersion.toInt() * 100) + devVersion.toInt()
```

Comment:
ReVanced Manager v2 will not use the versionCode calculator that v1 uses, it was a mistake 

The right way is to bump by 1 for each build